docs/docs/sourcing-from-buttercms.md

```diff
@@ -301,7 +295,7 @@ To Pull down content into gatsby run:
 gatsby develop
 ```
 
-### Testing with GrapiQl
+#### Testing with GrapiQl
```

Comment:
```suggestion
### Testing with GraphiQL
```
I'd keep this as an h3, as it could refer to the whole document. GraphiQL was also misspelled.